sample-android-app/build.gradle

```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
```

Comment:
Don't jetify appcompat.
Check `Deps` file, change `Deps.appcompat.XXX` artefacts to `Deps.androidx.XXX` and use them in `build.gradle` of sample app.

Reply:
Alright, will do right away.

Reply:
@CherryPerry I also changed `Support` to `AndroidX` and `support` to `androidx` in `Deps` file for clarity.